.travis.yml

```diff
@@ -1,23 +1,30 @@
-language: c
+language: python
```

Comment:
Normally we use C since we use conda to set up the dependencies anyway - can you change this back?

Reply:
Sure.